eval/python/word_analogy.py

```diff
@@ -71,8 +71,9 @@ def distance(W, vocab, ivocab, input_term):
 if __name__ == "__main__":
     N = 100;          # number of closest words that will be shown
     W, vocab, ivocab = generate()
+    input_message = "\nEnter three words (EXIT to break): "
     while True:
-        input_term = raw_input("\nEnter three words (EXIT to break): ")
+        input_term = raw_input(input_message) if sys.version_info < (3, 0) else input(input_message)
```

Comment:
how about just `sys.version_info.major` although i'm not even sure it's worthwhile keeping around the python2 support

Reply:
Yeah, thanks! That's cleaner 🙌

Actually, the other 2 python scrips were added 2 years ago with no support to Python 2: https://github.com/stanfordnlp/GloVe/blob/3776e82961ac23e339fe7f1b70ddc8119077a7a7/eval/python/distance.py#L71

I made it consistent with other scripts...